components/eventing-controller/pkg/backend/jetstreamv2/stubs_test.go

```diff
@@ -0,0 +1,222 @@
+package jetstreamv2 //nolint:nolintlint,testpackage
```

Comment:
the problem is, that we the pull-lint jobs still fails due to this

Reply:
I actually **like** the **recommendation** from `nolintlint` to give a **reason** why a linter is ignored because otherwise it is impossible to judge by looking at the code without running the linter:

```go
pkg/backend/jetstreamv2/jetstream_internal_unit_test.go:1:21: directive `//nolint:testpackage` should provide explanation such as `//nolint:testpackage // this is why` (nolintlint)
package jetstreamv2 //nolint:testpackage
pkg/backend/jetstreamv2/stubs_test.go:1:21: directive `//nolint:testpackage` should provide explanation such as `//nolint:testpackage // this is why` (nolintlint)
```

Reply:
This is a recommendation from linter in the IDE.